res/i18n/fr/labels.json

```diff
@@ -470,7 +470,7 @@
 	"sc_boost": "SC boost",
 	"sc_boost:": "SC boost :",
 	
-	"sc_preamp_makeup:db_db": "SC Préamp,\nSurgain\n(dB, dB)",
+	"sc_preamp_makeup:db_db": "SC Préamp,\nGain\n(dB, dB)",
```

Comment:
I have never heard "Surgain" in the tech vocabulary. In this context, this stage add level, what is simply a "gain" in french.
So for me : "keep it simple" and "less is more" if it don't can be ambiguous.

Reply:
if that would be only a "gain", then it could have been the same in English.
@sadko4u : can you please tell us if that's only a "gain stage" or if that's more than that?